Makefile

```
@@ -74,6 +81,9 @@ docker-nbval:
 docker-clean:
 	$(DOCKER_RUN) make clean
 
+docker-binder-nbval:
+	docker run python4compscience-binder make nbval-native
```

```suggestion
	$(DOCKER_RUN) make nbval-native
```

Running `docker run python4compscience-binder make nbval-native` passes in the CI however fails (for me) locally saying that the make target does not exist.

This is because by default the CI docker executions mount the current directory (i.e. the project directory) and then the container starts within that directory, but the default working directory set in the Dockerfile is `/io` which is just empty, so when you execute just `docker run ... make nbval-native` it fails because none of the project files are there.

Looked through the make file and I think I remember solving this issue in other cases by defining `DOCKER_RUN` which mounts the required volumes in from the start, and then using `$(DOCKER_RUN) make ...` in the makefile so that the targets execute correctly within the container.

So if you want to be able to run the tests locally with just `make docker-binder-nbval` then this change is needed, along with the above change to the image name.